app.js

```diff
@@ -23,19 +23,25 @@ app.use('/', indexRouter);
 app.use('/users', usersRouter);
 
 // catch 404 and forward to error handler
-app.use(function(req, res, next) {
-  next(createError(404));
+app.use(function (req, res, next) {
+    next(createError(404));
 });
 
 // error handler
-app.use(function(err, req, res, next) {
-  // set locals, only providing error in development
-  res.locals.message = err.message;
-  res.locals.error = req.app.get('env') === 'development' ? err : {};
-
-  // render the error page
-  res.status(err.status || 500);
-  res.render('error');
+app.use(function (err, req, res, next) {
+    // set locals, only providing error in development
+    res.locals.message = err.message;
+    res.locals.error = req.app.get('env') === 'development' ? err : {};
+
+    // render the error page
+    res.status(err.status || 500);
+    res.render('error');
+});
+
+const port = 3000;
+
+app.listen(port, () => {
+    console.log('Server started on port: ' + port);
```

Comment:
The listener is already set up within [`bin/www`](https://github.com/jessicawong217/Advanced_Web_Dev_Project/blob/741495542cd58d13c2b52c380d341d32d0562deb/bin/www#L28)